README.md

```diff
@@ -9,6 +9,40 @@ A python package for efficient pickling.
 
 ## Installation
 
+```bash
+pip install slim-trees
+# or
+mamba install slim-trees -c conda-forge
+```
+
+## Usage
+
+Using `slim-trees` does not affect your training pipeline.
+Simply replace calls to `pickle.dumps` with `slim_trees.dump_sklearn_compressed`:
```

Comment:
```suggestion
Simply replace `pickle.dump` calls with `slim_trees.dump_sklearn_compressed`:
```

Reply:
Also, `dump_sklearn_compressed` does not have the same signature as `pickle.dump`.
`dump_sklearn_compressed` takes a path as argument while `pickle.dump` takes a file as argument. 
`slim_trees.sklearn_tree.dump_sklearn` would be a drop-in replacement for `pickle.dump`.

Reply:
Yes, let's include both maybe